crud/interface.go

```diff
@@ -48,6 +52,9 @@ type ResourceDeleter interface {
 	Delete() error
 }
 
+// }}}
+
+// TODO: doc
```

Comment:
"Some resources in the oracle API are removed asynchronously, so even after they claim to be gone, other dependencies haven't been notified of that fact. This facility allows us to add an artificial delay for resources that need a little time to let the oracle API backend catch up with reality."